blog/2015/04/instructor-debriefing-2015-04-14.html

```diff
@@ -0,0 +1,47 @@
+---
+layout: blog
+root: ../../..
+author: Kate Hertweck
+title: "2015 Post-Workshop Instructor Debriefing, Round 7"
+date: 2015-04-14
+time: "11:20:00"
+category: ["Teaching"]
+---
+<!-- start excerpt -->
+<p>
+  The mentorship team held our latest round of post-workshop debriefing sessions for instructors who taught recently. Instructors from workshops at Clemson University, University of Miami, University of Melbourne, University of Oklahoma Libraries, Harvard School of Public Health, and Weill Cornell Medical College joined us for our discussions, as well as a few new instructors who will be teaching in coming weeks. Here's a recap of common themes and highlights:
+</p>
+<!-- end excerpt -->
+<b>Things that worked well:</b>
+<ul>
+	<li>
+	<i>Teaching R and Python together:</i> University of Melbourne ambitiously covered Unix, Git, SQL, Python and R over a three day workshop. Although they reported some learning fatigue near the end, the lesson pacing was appropriate and they largely positive feedback feedback. R and RStudio was taught first, followed by Python and a short introduction to Jupyter Notebook, which allowed instructors to highlight parallels between the scripting language and their respective interfaces.
```

Comment:
Is "they largely positive feedback feedback." correct? The duplicate "feedback" sounds a little strange to me.